.github/workflows/tox.yaml

```diff
@@ -0,0 +1,89 @@
+name: Tox
+on:
+  push:
+    branches:
+      - "main"
+      - "snapcraft/7.0"
+      - "release/*"
+      - "hotfix/*"
```

Comment:
What about `feature/*` for long-lived feature branches?  (I'm thinking of the `feature/kernel` branch)

Reply:
I can definitely add whatever we want here - I just copied this from `tests.yaml`.

`feature/*` is probably something we'd want in general on all our projects, so it might be better to have that discussion next week when I create the template repo?

Reply:
Sounds good!